### ISSUE TITLE
LIKE as Sqlizer

### ISSUE THREAD

**Issue by limoli** [Masterminds/squirrel#122]
I need to add dynamically a **LIKE condition** to a **squirrel.And condition**.
I cannot write simply a raw LIKE query because I need to append it in a conditions array (of Sqlizer interfaces).

I am looking something like this:
```
squirrel.Eq{
    "name" : "%irrel",
}
```
Or
```
squirrel.Like{
    "name" : "%irrel",
}
```
Is it already implemented?
I think that it could be very useful.

**Comment by lann** [comment]
The Expr function can be used for this:

```go
sq.Expr("name LIKE ?", "%irrel")
```

**Comment by Limoli** [comment]
I forked it creating **Lk** and **NotLk** methods. I think it would be very useful to merge these methods in the v1 branch.

**Comment by lann** [comment]
Fixed by #123
